requirements-cuda.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs-material
+fastapi[standard]
```

Comment:
Should we go with fast-api or something else ? Something faster maybe? What are the alternatives to this.

Reply:
I thing the repo should be focus more on python module @shauray8

Reply:
We can have multple repos which will be wrapper or we can say server templates for this package.

Reply:
like for serverless implementation .. repo will be differnt .. something like that.. or maybe we can do that in this repo also